test_config.yaml

```diff
@@ -0,0 +1,36 @@
+platforms:
+  override_default_platforms: false
+  increased_platform_scope: false
+
+levels:
+  - name: samples
+    description: >
+      Prepare samples.
```

Comment:
Is it intentional to have a newline after the `>` ?

Reply:
This is the schema used in NRF test_config, so I just used the same format.